Formula/ctop.rb

```diff
@@ -0,0 +1,24 @@
+class Ctop < Formula
+  desc "Top-like interface for container metrics"
+  homepage "https://bcicen.github.io/ctop/"
+  url "https://github.com/bcicen/ctop/archive/v0.4.1-deps.tar.gz"
```

Comment:
This `-deps` confuses the version parser; please add the version below this line using `version "0.4.1"`.

```
Error: 1 problem in 1 formula
ctop:
  * Stable: version (deps) is set to a string without a digit
```